docker-compose.yml

```diff
@@ -33,11 +35,13 @@ services:
         no_proxy:
         TF_ANNOTATION: "no"
         USER: "django"
-        DJANGO_CONFIGURATION: "production"
+        DJANGO_CONFIGURATION: "staging"
```

Comment:
The default configuration should be production because users want to run it without modifications by default. Staging and debugging configurations only for local development. Should be inside docker-compose.override.yml.